R/normalize-sf.R

```diff
@@ -23,44 +23,46 @@ pointData.sfc_POINT <- function(obj) {
   )
 }
 
+#' @export
+pointData.POINT <- function(obj) {
+  check_crs(obj)
+
+  bbox <- sf_bbox(obj)
```

Comment:
I have to do this because I may turn it into a matrix below and that drops the bbox